### ISSUE TITLE
[Feature Request]  give a example of using FlowInvoker to do streaming answers

### ISSUE THREAD

**Issue by liyuming1978** [microsoft/promptflow#1213]
**Is your feature request related to a problem? Please describe.**
package flowpromt to app show how to use FlowInvoker  to make a flow as a function call, but if a llm model is in streaming mode (answer with split text in one function call),  would you please give a sample for it?

**Describe the solution you'd like**
maybe a sample only with a "fake" llm would be more welcome to make starter to quickly know how to use promptflow.
for ex.  a fake llm only give a fix response or several fix streaming reponse.

**Comment by liyuming1978** [comment]
thanks so much  @D-W-

**Comment by liyuming1978** [comment]
from promptflow import load_flow
from promptflow._sdk._serving.flow_invoker import FlowInvoker
all load flow as a function,  I find "FlowInvoker" is used at http-server, example is "package your flow as a local app"

**Comment by D-W-** [comment]
Yes, `load_flow` itself will load flow from file and return a Flow object. The object is callable and can be used like a function.
`FlowInvoker` works in operations level, it can take a flow to initialize and provides functionality to execute it.
The main difference between them is `load_flow` is a public API and `FlowInvoker` is a private class. 
We also plan to merge implementation for them and in future the call stack will be like this:
`load_flow -> __call__ -> FlowInvoker.invoke`

So in summary, it's recommended to use `load_flow`.
